projects/packages/forms/src/dashboard/components/layout/style.scss

```diff
@@ -14,28 +14,63 @@ body.toplevel_page_jetpack-forms {
 	padding: 0;
 	width: 100%;
 
-	&.jp-forms__landing {
+	.jp-forms__logo-wrapper,
+	.jp-forms__layout-header {
+		flex: 0 1 auto; /* Flex, but don't grow */
+	}
 
-		.jp-forms__logo {
-			margin: 40px 64px;
-			width: 330px;
+	.jp-forms__logo {
+		margin: 20px 48px;
+		width: 200px;
 
-			@media (max-width: 660px) {
-				margin: 40px 24px;
-			}
+		@media (max-width: 660px) {
+			margin: 20px 24px;
 		}
 	}
+}
+
+#jp-forms-dashboard .jp-forms__layout {
+
+	.jp-forms__dashboard-tabs {
+
+		& > .components-tab-panel__tabs {
+			margin: 0;
+			padding: 0 48px;
+			border-bottom: 1px solid #dcdcde;
 
-	&.jp-forms__inbox {
+			.components-tab-panel__tabs-item {
+				border-bottom: 2px solid transparent;
+				font-size: var(--jp-forms-font-size-regular);
+				font-weight: 400;
+				height: 40px;
+				padding: 8px 16px;
 
-		.jp-forms__logo {
-			margin: 20px 48px;
-			width: 200px;
+				&.is-active {
+					border-color: var(--jp-green-50);
+					font-weight: 500;
+
+					&::after {
+						display: none;
+					}
+				}
+			}
+
+			@media (max-width: 600px) {
+				top: 0;
+			}
 
 			@media (max-width: 660px) {
-				margin: 20px 24px;
+				padding: 0 24px;
 			}
 		}
+
+		.components-tab-panel__tab-content {
+			background-color: var(--jp-forms-white-off);
+		}
+	}
+	// Temporary to adjust spacing for export button
```

Comment:
Maybe it is better to use "TODO" so we can find it later more easily? Unless by temporary you mean it should be very short lived

Reply:
Updated.